### ISSUE TITLE
dependencies

### ISSUE THREAD

**Issue by firstvanq** [TheLastBen/fast-stable-diffusion#2054]
![image](https://user-images.githubusercontent.com/76738608/234940745-395be30a-06cf-4241-a161-d8515c324471.png)
hey there, yesterday, the notebook was working fine, but today it is showing this error, after installing the module, it keeps on saying different modules not found. idk what the problem is, but there might be some requirements/dependencies missing in the code if it was updated recently?

**Comment by HelpANewbie** [comment]
yes looks like an update is required - I am getting the same problem

**Comment by DevArqSangoi** [comment]
The problem is on Requirements cell, it isn't working, maybe Colab problem. I'm trying a workaround here until TheLastBen try fix it.

**Comment by TheLastBen** [comment]
fixed, update the notebook

**Comment by cosmicoxytocin** [comment]
Can confirm, everything works as intended now👍

**Comment by thebrodyaga** [comment]
Something went wrong
ModuleNotFoundError: No module named 'onnxruntime.capi.onnxruntime_pybind11_state'

**Comment by TheLastBen** [comment]
@thebrodyaga which extension is causing it?

**Comment by firstvanq** [comment]
yayyy, thanks ben!!! i gotta sleep now, but will for sure run it tomorrow!

**Comment by TheLastBen** [comment]
I'll check it out
